client/web/src/components/SelfHostedCta.story.tsx

```diff
@@ -0,0 +1,16 @@
+import { storiesOf } from '@storybook/react'
+import React from 'react'
+
+import { SelfHostedCta } from './SelfHostedCta'
+import { WebStory } from './WebStory'
+
+const { add } = storiesOf('web/SelfHostedCta', module).addDecorator(story => (
```

Comment:
Let's move marketing stories into their own Storybook folder: 

```suggestion
const { add } = storiesOf('web/markering/SelfHostedCta', module).addDecorator(story => (
```